### ISSUE TITLE
Search doesn't work in Firefox

### ISSUE THREAD

**Issue by RyanCarey** [tog22/eaforum#40]
Hi! :)

The forum's search option doesn't work when I'm using Mozilla Firefox (see apendix). It works in Chrome and in Internet Explorer.

Best regards,
Luís Campos

**Comment by LLCampos** [comment]
![ea_problem](https://cloud.githubusercontent.com/assets/12008784/10007398/b7683a10-60bc-11e5-922f-97dbcd4375b2.PNG)

This is what i get.

**Comment by tog22** [comment]
I had one of the tech team look into this but with no progress. Will update
if they make any.

**Comment by peterhurford** [comment]
@tog22 Who's looking at it?

**Comment by arichard4** [comment]
Not me, but:
Reproduced. Of note is that, at least for the search term 'pesticides', it does display a google ad for pesticides, but no actual content.

**Comment by arichard4** [comment]
Haven't tested it yet, but one potential solution to this is here:
https://productforums.google.com/forum/#!topic/customsearch/lQAKjUmysfo

tl;dr: We need to remove the extra parentheses provided in Google's default search javascript:

``````
<script>
(function() {
var cx = '009626762821654294175:32pjjkjpoxk';
var gcse = document.createElement('script');
gcse.type = 'text/javascript';
gcse.async = true;
gcse.src = (document.location.protocol == 'https:' ? 'https:' : 'http:') +
'//www.google.com/cse/cse.js?cx=' + cx;
var s = document.getElementsByTagName('script')[0];
s.parentNode.insertBefore(gcse, s);
})();
</script>
<gcse:searchresults-only></gcse:searchresults-only>```
``````

**Comment by patbl** [comment]
I can't reproduce this. Maybe it was fixed along with #45. Does anyone else have problems with search in Firefox?

**Comment by LLCampos** [comment]
Yes, it's fixed. (I was the one who reported the problem)
